src/rules/style.ts

```diff
@@ -43,6 +43,8 @@ export default {
   'no-consecutive-blank-lines': [true, 2],
   // Disallows parameter properties in class constructors.
   'no-parameter-properties': false,
+  // Don’t if you import “foo” anyway.
```

Comment:
lol wtf. My guess would be that it means to not use a `/// reference` comment for things that are already imported. I would just turn it on for now and we can disable if it is doing something unexpected.